test/01_type/sec_to_time_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test/unit'
+require_relative '../../src/01_type/sec_to_time'
+
+class TypeTest < Test::Unit::TestCase
+  def test_sec_to_time_cond1
+    actual = round_ones_place(46_979)
```

Comment:
Keep in mind.
<img width="300" alt="2017-04-06 19 23 37" src="https://cloud.githubusercontent.com/assets/1853727/24749592/98de6526-1afe-11e7-8763-60d9054e07f3.png">